src/utils/components/CloudinitModal/CloudInitNetworkForm.tsx

```diff
@@ -54,6 +53,20 @@ export const CloudinitNetworkForm: React.FC<CloudinitNetworkFormProps> = ({
             />
           </FormGroup>
           <FormGroup
+            labelHelp={
+              <HelpTextIcon
+                bodyContent={
+                  <div>
```

Comment:
This would look cleaner as a `<Trans>` block. You can use a `<br />` tag in between the lines. Let me know if you need help.